OpenSim/Simulation/Control/Controller.h

```diff
@@ -44,6 +44,13 @@ class Actuator;
  * The defining method of a Controller is its computeControls() method.
  * @see computeControls()
  *
+ * @note Version 4.0 introduced the ability to have multiple components
+ * with the same name as long as the components have unique absolute paths
+ * (i.e., they are located in different components). However, Controllers 
+ * will currently not work properly if you have multiple actuators with
```

Comment:
Could say that `Controller` is not using the `Socket` mechanism to locate and connect to dependent Actuators, so for the time-being it doesn't support having multiple actuators with the same name.